### PR TITLE
[Partial] Builds LTE controller rock

### PR DESCRIPTION
**Pull request by gruyaume**
PR to build the LTE controller rock. This rock will be used by the following services in Magma's Orchestrator bundle:
- ha
- lte
- policydb
- smsd
- subsriberdb
- subscriberdb-cache

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ platforms:
 cmd: ["lte", "-run_echo_server=true", "-logtostderr=true", "-v=0"]
 
 env:
-  - DATABASE_SOURCE: f"dbname=magma user=username password=password host=127.0.0.1 sslmode=disable"
+  - DATABASE_SOURCE: "dbname=magma user=username password=password host=172.17.0.2 sslmode=disable"
```
Comment (gruyaume) [inline rockcraft.yaml]: I'd keep this to `127.0.0.1`, it's strange for it to be a very specific address

File: tests/integration/test_integration.py [hunk 2]
```diff
@@ -1,28 +1,140 @@
...
+    def _get_image_name_and_version() -> tuple:
+        """Fetches image name and version from rockcraft.yaml."""
+        with open(ROCKCRAFT_FILE_PATH, "r") as file:
+            data = yaml.safe_load(file)
+            image_name = data["name"]
+            version = data["version"]
+        return (image_name, version)
+
+    @property
+    def _host_ip(self) -> str:
```
Comment (gruyaume) [inline tests/integration/test_integration.py]: I don't think it makes sense for `_host_ip` to be a property of the `TestLTEControllerRock` class. To make it simple I'd simply use a method called `_get_container_ip(name: str) -> str`.

File: rockcraft.yaml (hunk 1 above)
Comment (sanchezfdezjavier) [reply rockcraft.yaml]: Changed.

File: tests/integration/test_integration.py (hunk 2 above)
Comment (sanchezfdezjavier) [reply tests/integration/test_integration.py]: Done.

**Review by ghislainbourgeois — APPROVED** [review]

**Pull request merged**